quickstarts/maven/wildfly-jar/pom.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2020 Red Hat, Inc.
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at:
+
+        https://www.eclipse.org/legal/epl-2.0/
+
+    SPDX-License-Identifier: EPL-2.0
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.eclipse.jkube.quickstarts.maven</groupId>
+  <artifactId>wildfly-jar</artifactId>
+  <version>1.0.0-alpha-4</version>
+  <name>Eclipse JKube :: Quickstarts :: Maven :: Wildfly JAR</name>
+  <packaging>war</packaging>
+
+  <properties>
+    <version.wildfly.jar>2.0.0.Alpha6</version.wildfly.jar>
+    <version.wildfly>20.0.0.Final</version.wildfly>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <failOnMissingWebXml>false</failOnMissingWebXml>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <!-- crash image name -->
+    <!--<jkube.generator.name>%a/%g:%l</jkube.generator.name>-->
```

Comment:
Is this showcasing something or is it dead code?

Reply:
Dead code, removed.